bin/makePsfLibrary.py

```
@@ -156,7 +156,12 @@ def makeSubLibrary(images, catname, output_file, fout, psfSize=33):
                         hdu = pyfits.hdu.image.PrimaryHDU()
                     else:
                         hdu = pyfits.hdu.image.ImageHDU()
-                    hdu.data = warpedPsf.computeImage().getArray()
+                    try:
+                        hdu.data = warpedPsf.computeImage().getArray()
+                    except:
```

Bare except is bad form.  Should name the exception you are expecting to see.